### DISCUSSION TITLE
Una pregunta (TEAMS)

### DISCUSSION THREAD

**Discussion by AlexOrrala** [Launch-X-Latam/MisionFrontEnd#462] [Team Posts]
He tenido problemas a la hora de unirme al canal de teams llevo varios días consultando y no recibo respuesta. hay algún modo de contacto directo para problemas cómo el que tengo?

**Comment by mariocanul30** [comment]
contactalos por instagram en @innovaccionvirtual y les mandas un dm para decirles acerca de tu problema

**Comment by AlexOrrala** [comment]
> contactalos por instagram en @innovaccionvirtual y les mandas un dm para decirles acerca de tu problema

Listo, muchas gracias.

**Comment by Strato1257** [comment]
Si gustas escríbeme! y vemos como podemos ayudarte

**Comment by AlexOrrala** [comment]
> Si gustas escríbeme! y vemos como podemos ayudarte

sisisi, claro que si, estoy un poco retrasado supongo y estoy buscando la forma de poder ingresar rápido para poder entra la corriente si tienes alguna forma de contacto me ayudaría mucho, muchas gracias.

**Comment by lilianaghr12** [comment]
yo estoy igual, no puedo visualizar el equipo en Teams y ya escribí a innovaccion, te dieron alguna solución?

**Comment by AlexOrrala** [comment]
> yo estoy igual, no puedo visualizar el equipo en Teams y ya escribí a innovaccion, te dieron alguna solución?

La verdad es que no, envié un mensaje al correo y me respondieron pero la respuesta no es rápida e instantánea. Escribí todos los detalles hace 3 días y hace 1 día me respondieron que cual era el problema. Se lo reescribí y estoy esperando la respuesta.

**Comment by Elli0t77** [comment]
Estoy igual, hace como un mes les escribí y nada, ví por ahí que por Instagram y aún así no responden. ¿Hay alguien que por favor me pueda ayudar?
